src/index.html

```diff
@@ -22,6 +22,340 @@
+        <div class="card__rate">
+          <div class="card__rate-stars">
+            <div class="stars stars--4 stars--card">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+          </div>
+          <div class="card__rate-count">Reviews: 5</div>
+        </div>
+        <div class="card__price">
+          <div class="card__price-title">Price:</div>
+          <div class="card__price-value">$2,199</div>
+        </div>
+        <button class="card__buy-button">
+          <a
+            href="#"
+            class="card__buy-button-link"
+          >
+            Buy
+          </a>
+        </button>
+      </div>
+      <div class="card">
+        <div class="card__image"></div>
+        <div class="card__info">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </div>
+        <div class="card__code">Product code: 195434</div>
+        <div class="card__rate">
+          <div class="card__rate-stars">
+            <div class="stars stars--4 stars--card">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+          </div>
+          <div class="card__rate-count">Reviews: 5</div>
+        </div>
+        <div class="card__price">
+          <div class="card__price-title">Price:</div>
+          <div class="card__price-value">$2,199</div>
+        </div>
+        <button class="card__buy-button">
+          <a
+            href="#"
+            class="card__buy-button-link"
+          >
+            Buy
+          </a>
+        </button>
+      </div>
+      <div class="card">
+        <div class="card__image"></div>
+        <div class="card__info">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </div>
+        <div class="card__code">Product code: 195434</div>
+        <div class="card__rate">
+          <div class="card__rate-stars">
+            <div class="stars stars--4 stars--card">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+          </div>
+          <div class="card__rate-count">Reviews: 5</div>
+        </div>
+        <div class="card__price">
+          <div class="card__price-title">Price:</div>
+          <div class="card__price-value">$2,199</div>
+        </div>
+        <button class="card__buy-button">
+          <a
+            href="#"
+            class="card__buy-button-link"
+          >
+            Buy
+          </a>
```

Comment:
Using an anchor tag inside a button is not semantically correct. Consider using either a button or an anchor tag, but not both. If you need a button that behaves like a link, you can style an anchor tag to look like a button.